opentelemetry-prometheus/tests/integration_test.rs

```diff
@@ -6,6 +6,33 @@ use opentelemetry::{
 use opentelemetry_prometheus::PrometheusExporter;
 use prometheus::{Encoder, TextEncoder};
 
+#[test]
+fn free_unused_instruments() {
```

Comment:
Hmm for some reason I run this test against `master` and it works. Not sure if I missed something.

Reply:
This behavior seems to have changed between dashmap version `4.0.0-rc6` and dashmap version `4.0.0` which was released this week. If you cargo update you should see the issue.

Reply:
Ha make sense.  LGTM